modules/floating-buttons/assets/js/editor/hooks/ui/editor/document/select.js

```diff
@@ -0,0 +1,34 @@
+import After from 'elementor-api/modules/hooks/ui/after';
```

Comment:
@louiswol94 we discussed this on the other PR, at the time I was deselecting when a container was selected, now I am selecting the internal element when a container is selected, so that when you insert a Floating Button from the remote library automatically the widget controls are shown. This has been tested yesterday / today from the elementor team and I think that the condition is ok as it applies only to the document type for floating buttons.
As you can see below this only applies to sections and containers so there should not be infinite recursion, and in this document you should never have anything more than a container and a widget inside it.